modules/dnn/src/darknet/darknet_io.cpp

```diff
@@ -508,13 +502,9 @@ namespace cv {
                 return true;
             }
 
-
-            bool ReadDarknetFromWeightsFile(const char *darknetModel, NetParameter *net)
+            bool ReadDarknetFromWeightsStream(std::istream &darknetModelStream, NetParameter *net)
```

Comment:
Do you mind just rename `darknetModelStream` to `ifile` here?

Reply:
Of course. I renamed.